lib/Echidna/Types/Corpus.hs

```diff
@@ -1,6 +1,9 @@
 module Echidna.Types.Corpus where
 
-import Data.Set (Set)
+import Data.Set (Set, size)
```

Comment:
`length` is defaulted to `size`

https://hackage.haskell.org/package/containers-0.6.4.1/docs/src/Data.Set.Internal.html#line-317

semantics up to you